src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -799,13 +799,15 @@ db::DataBox<tmpl::list<Tags...>>::reset_compute_items_after_mutate(
     tmpl::list<ComputeItemsToReset...> /*meta*/) noexcept {
   EXPAND_PACK_LEFT_TO_RIGHT(add_reset_compute_item_to_box<ComputeItemsToReset>(
       detail::get_argument_list<ComputeItemsToReset>{}));
-
-  using next_compute_tags_to_reset =
-      tmpl::transform<tmpl::append<tmpl::filter<
-                          typename DataBox<tmpl::list<Tags...>>::edge_list,
-                          std::is_same<tmpl::pin<ComputeItemsToReset>,
-                                       tmpl::get_source<tmpl::_1>>>...>,
-                      tmpl::get_destination<tmpl::_1>>;
+  using current_tags_to_reset = tmpl::list<ComputeItemsToReset...>;
```

Comment:
Should this also contain the tags reset in previous iterations of this reset procedure?

Reply:
Yes it should in order to be more efficient, but it is not necessary.  This function will be changed in my next PR, so if it is okay, I'd prefer to make this improvement then

Reply:
OK.